tests/org.pitest.pitclipse.tests.coverage.report/pom.xml

```diff
@@ -43,25 +43,25 @@
 		<dependency>
 			<groupId>org.pitest</groupId>
 			<artifactId>org.pitest.pitclipse.runner</artifactId>
-			<version>2.1.0</version>
+			<version>2.1.2-SNAPSHOT</version>
```

Comment:
To keep things maintainable, I would use `${project.version}` instead of explicit version numbers